src/server.js

```diff
@@ -22,47 +23,82 @@ const reading = {
   name: "reading philosophy books"
 };
 
+const allSkills = [donutEating, saxophonePlaying, reading];
+
 const userSkill = (skill, regnalNumber) => ({
   skill,
   level: Math.pow(2, regnalNumber)
 });
 
-const getSkills = (name, regnalNumber) => {
-  const skills =
-    name.toLowerCase() === "lisa" ? [saxophonePlaying, reading] : [donutEating];
-
-  return skills.map(skill => userSkill(skill, regnalNumber));
-};
+const userSkills = (skills, regnalNumber) =>
+  skills.map(skill => userSkill(skill, regnalNumber));
 
 const computeRegnalNumber = name =>
-  users.reduce(
-    (count, user) => (user.firstName === name ? count + 1 : count),
+  Object.keys(users).reduce(
+    (count, userId) => (users[userId].firstName === name ? count + 1 : count),
     1
   );
 
-const createUser = ({ firstName, lastName }) => {
+const createUser = ({ firstName, lastName, skills }) => {
   const regnalNumber = computeRegnalNumber(firstName);
+  const id = `user-${userCount}`;
```

Comment:
You have 2 sources of truth for user count. I would rather use `Object.keys(users).length`.

Reply:
Makes sense - updated